docs/authorization.md

```diff
@@ -164,6 +164,69 @@ parent workspace.
 
 Service accounts declared within a workspace don't have access to initializing workspaces.
 
+## API binding authorizer
+
+If the requested resource type is part of an API binding, then the API binding authorizer verifies that
+the request is not exceeding the maximum permission policy of the related API export.
+Currently, the "local policy" maximum permission policy type is supported.
+
+### Local policy
+
+The local maximum permission policy delegates the decision to the RBAC of the related API export workspace.
```

Comment:
```suggestion
The local maximum permission policy delegates the decision to the RBAC of the related API export. To distinguish between local RBAC role bindings in that workspace and those for this these maximum permission policy, every name and group is prefixed with `apis.kcp.dev:binding:`.
```

Reply:
it is mentioned later but adding here additionally makes sense 👍 fixed